packages/vite-plugin-svelte/package.json

```
@@ -7,18 +7,18 @@
     "src"
```

would `types` have to be added here ? 
```suggestion
    "src",
    "types"
```